nav2_smac_planner/include/nav2_smac_planner/smac_planner_hybrid.hpp

```diff
@@ -89,6 +89,30 @@ class SmacPlannerHybrid : public nav2_core::GlobalPlanner
     const geometry_msgs::msg::PoseStamped & goal,
     std::function<bool()> cancel_checker)  override;
 
+  /**
```

Comment:
Get rid of these 4x getters, they're unused

Reply:
they are used actually?, some are used in testing.

Reply:
I use them in tests

Reply:
I believe you introduced the Wrapper class on in the tests to get access to protected members already. I think you can do that here as well and move the getters over into the test wrapper. I don't typically like to change source code to make testing easier; usually that's a queue to me that either we need to (1) rearchitect the code to be easier to test or (2) make testing fixtures to access internal data

Reply:
understood.